packages/db/prisma/schema.prisma

```diff
@@ -77,10 +77,10 @@ model User {
     payments                 Payment[]
     preference               Preference?
     membership               Membership  @default(FREE)
-    paidUntil                Int? // in milliseconds
+    paidUntil                DateTime?
     stripeCustomerId         String?
     stripePlanId             String?
```

Comment:
what does this field represent?

Reply:
membership is for `PRO` and `PREMIUM`. (Currently we only have `PRO` plan)

`stripePlanId` and `membershipPlan` store the product id, i.e 1 month subscription or 12 month subscription